neo4j/data-cron-loader/scripts/load-data-to-neo4j.sh

```diff
@@ -56,7 +56,7 @@ if [ -p /dev/stdin ]; then
     done < /dev/stdin
 fi
 
-for file in "$@"
+for file in $(ls $@)
```

Comment:
C'est une bonne idée. J'y avais pas pensé !

J'ai remarqué une erreur dans mon implémentation initiale. On ne distingue pas dossiers et fichiers : un dossier intitulé *.cypher serait actuellement accepté par le script :sweat_smile:

```suggestion
for file in $(find $@ -maxdepth 1 -type f)
```

Reply:
mouhai je crois que tu pinailles :p

et de plus ça complexifie la lecture du code pour gérer des cas qu'on ne rencontrera pas
mais ok go, ce n'est qu'une étude :)

Reply:
> mouhai je crois que tu pinailles :p

Ne jamais faire confiance aux utilisateurs :children_crossing:  :laughing: